src/modules/dashboard/components/Vesting/Vesting.css

```diff
@@ -0,0 +1,47 @@
+.main {
```

Comment:
You've implemented according to the spec, but man the design looks horrible on large screens :)

![Screenshot from 2021-12-08 16-46-41](https://user-images.githubusercontent.com/1193222/145229836-837cb397-8c42-4911-ad0a-833ac2aa4d99.png)

_(there's nothing to do here, this is just me ranting)_